### ISSUE TITLE
It is not clear what parts of the request are logged

### ISSUE THREAD

**Issue by hansmbakker** [dotnet/AspNetCore.Docs#22356]
There is a section on [Logging being built in to IHttpClientFactory clients](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-5.0#logging) but it is not clear what exactly is being logged.

Only the header information is explicitly mentioned (that you need to enable the trace log level for it).

What about the request / response bodies and other parts of the requests/responses?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 881cef77-11ef-ac11-06d0-0996a1ccfa1b
* Version Independent ID: 3d356334-e672-7e8b-1fef-7944ebee99eb
* Content: [Make HTTP requests using IHttpClientFactory in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests?view=aspnetcore-5.0)
* Content Source: [aspnetcore/fundamentals/http-requests.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/http-requests.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @stevejgordon
* Microsoft Alias: **scaddie**

**Comment by Rick-Anderson** [comment]
It states
**may include changes to request headers or to the response status code**

**Comment by hansmbakker** [comment]
Yes, and an example of how the logging would look like would help, and it would help to make explicit whether request/response bodies can be logged.

And if so, how.

**Comment by Rick-Anderson** [comment]
Thank you for contacting us. Due to a lack of activity on this discussion issue we're closing it in an effort to keep our backlog clean. If you believe there is a concern related to the ASP.NET Core which hasn't been addressed, please file a new issue.

**Comment by hansmbakker** [comment]
@Rick-Anderson 

I believe I asked a clear question - I didn't want to unnecessarily bump the discussion only to prevent the issue getting closed.

Can you please reopen the issue and give feedback on whether
- request/response bodies can be logged?
- whether a more explicit example of what a log item looks like could be added (to quickly assess whether the provided log detail is sufficient for people first using it)
